LINUX/io.github.horsicq.detect-it-easy.metainfo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 horsicq -->
+<component type="desktop-application">
+  <id>io.github.horsicq.detect-it-easy</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Detect It Easy</name>
+  <developer_name>horsicq</developer_name>
+  <update_contact>horsicq@gmail.com</update_contact>
+  <summary>Advanced file analyzer</summary>
+  <description>
+    <p>Detect It Easy, or abbreviated "DIE" is a program to analyze files you load into the application.</p>
+    <p>It detects, among other things, the compiler and linker used, signatures, and other information about files.</p>
+    <p>The program has been designed specifically to reveal file signatures and how they are packed.</p>
+    <p>Some of the binary formats supported:</p>
+    <ul>
+      <li>MS-DOS: DOS MZ</li>
+      <li>Windows: Portable Executable (PE)</li>
+      <li>*Nix: Executable and Linkable Format (ELF)</li>
+      <li>macOS: Mach Object (Mach-O)</li>
+    </ul>
+  </description>
```

Comment:
Let me know if this description of your app does it justice.

Reply:
The description is awesome. Thanks! :)